feature/login/src/main/java/com/epmedu/animeal/login/profile/presentation/FinishProfileScreen.kt

```diff
@@ -0,0 +1,33 @@
+package com.epmedu.animeal.login.profile.presentation
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.epmedu.animeal.extensions.currentOrThrow
+import com.epmedu.animeal.login.presentation.navigateToTabs
+import com.epmedu.animeal.navigation.navigator.LocalNavigator
+
+@Composable
+fun FinishProfileScreen(isFirstTime: Boolean = true) {
+    val viewModel: FinishProfileViewModel = hiltViewModel()
+    val navigator = LocalNavigator.currentOrThrow
+
+    FinishProfileScreenUi(
+        viewModel = viewModel,
```

Comment:
According to Google, you should not pass ViewModel instances down to other composables. Collect state on screen level and pass it to UI with callbacks.

Reply:
Done